RetailCoder.VBE/Settings/ConfigurationLoader.cs

```diff
@@ -113,7 +114,7 @@ public Configuration GetDefaultConfiguration()
         public ToDoMarker[] GetDefaultTodoMarkers()
         {
             var note = new ToDoMarker(RubberduckUI.ToDoMarkerNote, TodoPriority.Low);
-            var todo = new ToDoMarker(RubberduckUI.ToDoMarkerToDo, TodoPriority.Normal);
+            var todo = new ToDoMarker(RubberduckUI.ToDoMarkerToDo, TodoPriority.Medium);
```

Comment:
.....This is a breaking change. I don't know about this. I know we said we'd stuff all the breaking changes for the config into this release though, so... okay. We really need to treat the serialized classes like we would treat COM interfaces though. It's always okay to add to the interface, but never to change or remove from it.